draft-ietf-tls-svcb-ech.md

```diff
@@ -90,6 +90,84 @@ Origins that publish an "ech" SvcParam in their HTTPS record SHOULD also publish
 
 If all HTTPS records for an alt-authority contain "ech" SvcParams, the client MUST adopt SVCB-reliant behavior (as in {{disabling-fallback}}) for that RRSet.  This precludes the use of certain connections that Alt-Svc would otherwise allow, as discussed in {{Section 9.3 of !SVCB}}.
 
+# Examples
+
+~~~
+$ORIGIN simple.example. ; Simple example zone
+@ 300 IN A     192.0.2.1
+         AAAA  2001:db8::1
+         HTTPS 1 . ech=ABC...
+www 300 IN A 192.0.2.1
+           AAAA 2001:db8::1
+           HTTPS 1 . ech=ABC...
```

Comment:
I thought one of the points of the HTTPS record was that you didn have to keep manually adding A/AAAA sync ups between the apex and www. of a zone ?

Reply:
HTTPS records allow apex aliasing (as in the next example), but only for clients that support HTTPS records.  These examples mostly assume that compatibility with non-HTTPS-record-aware clients is required.

In this case, we could avoid duplicating the A/AAAA records by making `www` a CNAME to the apex (as in the multi-CDN example), but I thought that might not be the best choice for our simplest example.